solr/modules/ltr/src/test/org/apache/solr/ltr/TestLTROnSolrCloud.java

```diff
@@ -322,6 +323,43 @@ public void testSimpleQuery() throws Exception {
     assertEquals(result7_features, queryResponse.getResults().get(7).get("features").toString());
   }
 
+  @Test
+  public void testInterleaving() throws Exception {
+    SolrQuery query = new SolrQuery("{!func}sub(8,field(popularity))");
+    query.setRequestHandler("/query");
+    query.setFields("*,score,[interleaving]");
+    query.add(
+        "rq",
+        "{!ltr model=powpularityS-model model=powpularityS-model-for-interleaving reRankDocs=8}");
+
+    solrCluster
+        .getSolrClient()
+        .query(
+            COLLECTION,
+            query); // NullPointerException at LTRInterleavingTransformerFactory.java:103
+  }
```

Comment:
This test fails with NPE

Reply:
However, this branch(SOLR-16289) already fixed NPE. To reproduce the problem, checkout https://github.com/chlorochrule/solr/commit/33d1aa939a175310c1eb6f0b773a5c2d3a4ddb41